signalfx_lambda/tracing.py

```diff
@@ -20,6 +21,8 @@ def call(*args, **kwargs):
 
         try:
             with tracer.start_active_span(span_prefix + context.function_name, tags=span_tags) as scope:
+                scope.span.set_tag(ext_tags.SPAN_KIND, ext_tags.SPAN_KIND_RPC_SERVER)
```

Comment:
Why aren't you doing that line 18 above in `span_tags` ?

```python
span_tags[ext_tags.SPAN_KIND] = ext_tags.SPAN_KIND_RPC_SERVER
```

Reply:
Updated